tests/unit/amo/pages/TestLanguageTools.js

```diff
@@ -90,157 +81,129 @@ describe(__filename, () => {
     }),
   ];
 
-  function renderShallow({
-    i18n = fakeI18n(),
-    store = dispatchClientMetadata().store,
-    ...props
-  } = {}) {
-    return shallowUntilTarget(
-      <LanguageTools i18n={i18n} store={store} {...props} />,
-      LanguageToolsBase,
-    );
+  function render({ store = dispatchClientMetadata().store } = {}) {
+    return defaultRender({
+      history: createHistory({
+        initialEntries: [`/${lang}/firefox/language-tools/`],
+      }),
+      store,
+    });
   }
 
   it('renders LoadingText if language tools are not set', () => {
-    const root = renderShallow();
+    render();
 
-    expect(root.find(LoadingText)).not.toHaveLength(0);
+    expect(screen.queryAllByClassName('LoadingText').length).toBeGreaterThan(0);
   });
 
   it('renders LoadingText if language tools are empty', () => {
-    const { store } = dispatchClientMetadata();
+    const { store } = dispatchClientMetadata({ lang });
     store.dispatch(loadLanguageTools({ languageTools: [] }));
 
-    const root = renderShallow({ store });
+    render({ store });
 
-    expect(root.find(LoadingText)).not.toHaveLength(0);
+    expect(screen.queryAllByClassName('LoadingText').length).toBeGreaterThan(0);
   });
 
   it('renders language tools in your locale', () => {
     const { store } = dispatchClientMetadata({ lang: 'fr' });
     store.dispatch(loadLanguageTools({ languageTools }));
 
-    const root = renderShallow({ store });
+    render({ store, lang: 'fr' });
```

Comment:
This `lang` here isn't going to do anything, because `render` does not accept a `lang` argument. As it stands now, `render` is just using the global `lang` set at line 17, which happens to be `fr`, which is likely why this is working. When writing tests like this, it's probably safer to use a `lang` other than the default, to catch things like this.